### ISSUE TITLE
The executable  /Applications/Synergy.app/Contents/MacOS/synergys  could not be successfully started, although it does exist. Please check if you have sufficient permissions to run this program.

### ISSUE THREAD

**Issue by manfredinicolas** [symless/synergy-core#6643]
H, 

On start i have this message : The executable

/Applications/Synergy.app/Contents/MacOS/synergys

could not be successfully started, although it does exist. Please check if you have sufficient permissions to run this program.

**Comment by jzadra** [comment]
I have this as well.

**Comment by su37kk** [comment]
I am running into this failure as well on macOS Catalina. Does anyone have a fix?

**Comment by jzadra** [comment]
Yes. The fix is that yours software is not worth the trouble, and there is one called Sharemouse that is free, installs in one step, and works pretty well.

**Comment by smp4488** [comment]
I struggled with this and took the above advice and installed Sharemouse. It works so much better and is easier to configure.

**Comment by Jnewbon** [comment]
Try the step in [this guide](https://support.symless.com/hc/en-us/articles/360004804318-Error-codes)
If that doesn't work try contacting support.

**Comment by MrShashankSingh** [comment]
Try the RC build available here: https://snapshots.symless.com/public/1.11.1-rc2/
It worked for me on macOS Catalina.

**Comment by amaisano** [comment]
I have this issue too, but on Mac High Sierra.

**Comment by SeekingLight233** [comment]
> Try the RC build available here: https://snapshots.symless.com/public/1.11.1-rc2/
> It worked for me on macOS Catalina.

thx u!!!  It works on my machine :)

**Comment by temotodochi** [comment]
Tried to log in to get the RC, never got the login email from cloudflare. Gave up.

**Comment by andreacndn** [comment]
Is this still an issue with 1.13.1?

**Comment by avijit-pandey** [comment]
use sharemouse instead
ultra easy to use

**Comment by kamisake** [comment]
Thanks for the sharemouse tip, couldn't be any easier.

**Comment by Adrian-Dziubek** [comment]
> Yes. The fix is that this software is not worth the trouble, and there is one called Sharemouse that is free, installs in one step, and works pretty well.

It doesn't support Linux.
